### PR TITLE
Update release notes draft to version v1.26.0

### PR DESCRIPTION
**Pull request by ramrodo**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
This PR updates the Release Notes Draft to k/k v1.26.0

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:
This is an automated PR generated from `krel The Kubernetes Release Toolbox`

### CODE REVIEW

**Comment by leonardpahlke** [comment]
Thanks!
/lgtm
/hold for 1.26 to released

File: releases/release-1.26/release-notes/release-notes-draft.md [hunk 1]
```diff
@@ -100,11 +100,12 @@
...
 - Removed feature gates `ServiceLoadBalancerClass` and `ServiceLBNodePortControl`. These feature gates were enabled (and locked) since `v1.24`. ([#112577](https://github.com/kubernetes/kubernetes/pull/112577), [@andrewsykim](https://github.com/andrewsykim))
 - Reverted regression that prevented `client-go` latency metrics to be reported with a template URL to avoid label cardinality. ([#111752](https://github.com/kubernetes/kubernetes/pull/111752), [@aanm](https://github.com/aanm))
 - The `EndpointSliceTerminatingCondition` feature gate was graduated to GA. The gate is now locked and will be removed in v1.28. ([#113351](https://github.com/kubernetes/kubernetes/pull/113351), [@andrewsykim](https://github.com/andrewsykim))
+- [kubelet] Change default `cpuCFSQuotaPeriod` value with enabled `cpuCFSQuotaPeriod` flag from 100ms to 100µs to match the Linux CFS and k8s defaults. `cpuCFSQuotaPeriod` of 100ms now requires `customCPUCFSQuotaPeriod` flag to be set to work. ([#111520](https://github.com/kubernetes/kubernetes/pull/111520), [@paskal](https://github.com/paskal)) [SIG API Machinery and Node]
```
Comment (harshanarayana) [inline releases/release-1.26/release-notes/release-notes-draft.md:108]: `[kubelet] Changed default ` Minor change here.

Comment (ramrodo) [reply releases/release-1.26/release-notes/release-notes-draft.md:108]: Good catch! Thanks

**Comment by leonardpahlke** [comment]
PTAL @harshanarayana @AnaMMedina21 @laxmikantbpandhare @sayantani11
remove hold when ready

File: releases/release-1.26/release-notes/release-notes-draft.md [hunk 2]
```diff
@@ -216,7 +217,7 @@
 - Kubectl shell completions for the bash shell now include descriptions. ([#113636](https://github.com/kubernetes/kubernetes/pull/113636), [@marckhouzam](https://github.com/marckhouzam))
 - Kubernetes is now built with Go 1.19.1 ([#112287](https://github.com/kubernetes/kubernetes/pull/112287), [@palnabarun](https://github.com/palnabarun)) [SIG Release and Testing]
```
Comment (AnaMMedina21) [inline releases/release-1.26/release-notes/release-notes-draft.md:218]:
```suggestion
- Kubernetes is now built with Go `1.19.1` ([#112287](https://github.com/kubernetes/kubernetes/pull/112287), [@palnabarun](https://github.com/palnabarun)) [SIG Release and Testing]
```

**Review by AnaMMedina21 — COMMENTED** [review]
small suggestions

**Comment by ramrodo** [comment]
> small suggestions

Thanks for the suggestions @AnaMMedina21 
Suggestions addressed.

**Review by AnaMMedina21 — APPROVED** [review]
LGTM 🚀

**Comment by leonardpahlke** [comment]
/lgtm
@ramrodo feel free to remove hold when ready `/unhold`

**Comment by ramrodo** [comment]
Thanks, everyone!
/unhold

**Pull request merged**